### ISSUE TITLE
Trying to access array offset on value of type null

### ISSUE THREAD

**Issue by Fonta** [elementor/elementor#12228]
## Description
After updating, the admin dashboard shows html instead of the browser actually processing it.
It also shows a error at the bottom.
See screenshot: https://i.imgur.com/FNOaR5q.png

## Environment
<details>
<summary>System Info</summary>
```
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: mariadb.org binary distribution v5.6.49-89.0
	PHP Version: 7.4.6
	PHP Max Input Vars: 5000
	PHP Max Post Size: 128M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.5
	Site URL: https://www.x.nl
	Home URL: https://www.x.nl
	WP Multisite: No
	Max Upload Size: 128 MB
	Memory limit: 256M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: Europe/Amsterdam
	Debug Mode: Inactive

== Theme ==
	Name: Hestia Child
	Version: 0.1
	Author: ThemeIsle
	Child Theme: Yes
	Parent Theme Name: Hestia
	Parent Theme Version: 3.0.4
	Parent Theme Author: ThemeIsle

== User ==
	Role: administrator
	WP Profile lang: nl_NL
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/84.0.4147.135 Safari/537.36 Edg/84.0.522.63

== Active Plugins ==
	BackWPup
		Version: 3.7.1
		Author: Inpsyde GmbH

	Elementor
		Version: 3.0.0
		Author: Elementor.com

	Essential Addons for Elementor
		Version: 4.2.1
		Author: WPDeveloper

	Hummingbird
		Version: 2.5.1
		Author: WPMU DEV

	Invisible reCaptcha
		Version: 1.2.3
		Author: Mihai Chelaru

	Orbit Fox Companion
		Version: 2.9.16
		Author: Themeisle

	Rank Math SEO
		Version: 1.0.47.1
		Author: Rank Math

	Review Widgets for Google
		Version: 2.9
		Author: Trustindex.io <support@trustindex.io>

	Site Kit by Google
		Version: 1.14.0
		Author: Google

	Smush
		Version: 3.6.3
		Author: WPMU DEV

	User Role Editor
		Version: 4.56
		Author: Vladimir Garagulya

	WooCommerce
		Version: 4.4.1
		Author: Automattic

	WooCommerce Admin
		Version: 1.4.0
		Author: WooCommerce

	WooCommerce Menu Cart
		Version: 2.9.4.1
		Author: Jeremiah Prummer, Ewout Fernhout

	WP-ServerInfo
		Version: 1.66
		Author: Lester 'GaMerZ' Chan

	WPForms Lite
		Version: 1.6.2.2
		Author: WPForms


== Elements Usage ==
	
	page : 1
		button : 1
		google_maps : 1
		heading : 3
		icon-box : 2
		image-box : 3
		image-carousel : 1
		image-gallery : 1
		progress : 3
	wp-page : 16
		button : 1
		google_maps : 2
		heading : 39
		icon : 3
		icon-box : 5
		image : 15
		image-box : 10
		image-carousel : 1
		image-gallery : 1
		spacer : 56
		tabs : 1
		text-editor : 34
		wp-widget-wpforms-widget : 1


== Log ==
	
Log: showing 17 of 172020-04-21 17:48:13 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.9.7',
  'to' => '2.9.8',
)]
2020-06-06 15:34:51 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.9.8',
  'to' => '2.9.11',
)]
2020-06-17 19:25:10 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.9.11',
  'to' => '2.9.12',
)]
2020-07-23 22:52:34 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.9.12',
  'to' => '2.9.14',
)]
2020-08-25 18:11:16 [info] elementor::elementor_updater Started 
2020-08-25 18:11:16 [info] Elementor/Upgrades - _v_3_0_0_move_general_settings_to_kit Start  
2020-08-25 18:11:16 [info] Elementor/Upgrades - _v_3_0_0_move_general_settings_to_kit Finished 
2020-08-25 18:11:16 [info] Elementor/Upgrades - _v_3_0_0_move_default_colors_to_kit Start  
2020-08-25 18:11:16 [notice] System colors are disabled. nothing to do. 
2020-08-25 18:11:16 [info] Elementor/Upgrades - _v_3_0_0_move_default_colors_to_kit Finished 
2020-08-25 18:11:16 [info] Elementor/Upgrades - _v_3_0_0_move_saved_colors_to_kit Start  
2020-08-25 18:11:16 [info] Elementor/Upgrades - _v_3_0_0_move_saved_colors_to_kit Finished 
2020-08-25 18:11:16 [info] Elementor/Upgrades - _v_3_0_0_move_default_typography_to_kit Start  
2020-08-25 18:11:16 [notice] System typography is disabled. nothing to do. 
2020-08-25 18:11:16 [info] Elementor/Upgrades - _v_3_0_0_move_default_typography_to_kit Finished 
2020-08-25 18:11:16 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.9.14',
  'to' => '3.0.0',
)]
2020-08-25 18:11:16 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '2.9.14',
  'to' => '3.0.0',
)]

PHP: showing 1 of 1PHP: 2020-07-31 13:08:38 [notice X 10][/www/wp-content/plugins/elementor/includes/base/controls-stack.php::1451] Trying to access array offset on value of type null [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
```
</details>

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor v3.0.1

Feel free to update

Thanks!

**Comment by Fonta** [comment]
Indeed fixed. Thanks!
